databroker/_drivers/mongo_normalized.py

```diff
@@ -112,15 +115,21 @@ def __getitem__(self, name):
                     raise KeyError(f"No run with scan_id={N}")
         if run_start_doc is None:
             raise KeyError(name)
-        entry = self._doc_to_entry(run_start_doc)
+        uid = run_start_doc['uid']
+        try:
+            entry = self._cache[uid]
+            print('cache hit')
```

Comment:
This looks like it's from the commit of mine that you cherry-picked. That work-in-progress should be removed from this PR since it didn't immediately fix the performance issue you were investigating.

Reply:
Git tip: to remove a commit entirely, do

```
git rebase -i HEAD~SOME_NUMBER_OF_COMMITS_BACK
```

and then, in the list of commits that appears in the interactive view, simply delete the line corresponding to the commit you want to remove.

Reply:
updated